.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: ["*"]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust_version: [stable, "1.34.0"]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Rust toolchain
+      run: |
+        rustup default ${{ matrix.rust_version }}
+        rustup component add clippy rustfmt
+
+    - name: Build and test (default features)
+      run: |
+        cargo build --verbose
+        cargo test --verbose
+
+    - name: Build and test (all features)
+      run: |
+        cargo build --verbose --all-features
+        cargo test --verbose --all-features
+
+    - name: Build and test (no features)
+      run: |
+        cargo build --verbose --no-default-features
+        cargo test --verbose --no-default-features
+
+    - name: Rustfmt and Clippy
+      run: |
+        cargo fmt -- --check
+        cargo clippy
```

Comment:
This should have a `-D warnings` on it.

Outside of the scope of this PR, we should consider turning on more warnings similar to how I do it in my project https://github.com/BVE-Reborn/bve-reborn/blob/master/bve/src/lib.rs.

Reply:
Done! I had to use `cargo clippy -- -D warnings`, since `cargo clippy -D warnings` threw an error. Do you know if what I ended up adding is correct?

Reply:
Smells right to me! I forgot you needed the -- to give the argument to clippy and not to cargo directly.